pkg/cmd/clustertask/list.go

```diff
@@ -30,11 +30,15 @@ import (
 
 const (
 	emptyMsg = "No ClusterTasks found\n"
-	header   = "NAME\tDESCRIPTION\tAGE"
```

Comment:
We can probably keep `header` as a const and just print the `header` const below. WDYT?

Reply:
Yes, we can keep it constant but I did not see any benefit of using const or for that matter variable here so I removed it. But if you want I can revert this change.

Reply:
Added it back.